components/sup/src/manager/service/mod.rs

```diff
@@ -92,7 +92,7 @@ enum BindStatus<'a> {
 pub struct Service {
     pub service_group: ServiceGroup,
     pub bldr_url: String,
-    pub channel: String,
+    pub channel: String, // TODO: change to hcore::ChannelIdent
```

Comment:
Can we do this now, or was there some kind of gnarlier refactoring it entails?

Reply:
It seemed like a rabbit hole for questionable benefit. I'd like to get the initial change in and then try running down some of the TODOs so we can make the important progress (replacing all the misleading defaults) first.

Reply:
I sucked it up and just did it: https://github.com/habitat-sh/habitat/pull/6074/commits/cbebb204f2ee3b0b2cafe139cbf99ed1baf73d94#diff-320293bb5f68f709add66ac43f5ea52eR103